src/js/src/components/searchResults/PostSearchResults.vue

```diff
@@ -24,6 +24,14 @@
                  :result="result"
                  :rank-number="index" />
     </div>
+    <div class="pagingContainer">
+      <button :disabled="solrSettings.offset < 20" @click="getPreviousResults()">
+        Previous 20
+      </button>
+      <button :disabled="results.cardinality ? solrSettings.offset + 20 >= results.cardinality : solrSettings.offset + 20 >= results.numFound" @click="getNextResults()">
+        Next 20
+      </button>
+    </div>
```

Comment:
An enhancement here would be to implement a small pagination component so it is easier to move around and easier to change offset values etc.